packages/lib/src/components/ThreeDS2/components/Challenge/PrepareChallenge3DS2.tsx

```diff
@@ -60,10 +61,15 @@ class PrepareChallenge3DS2 extends Component<PrepareChallenge3DS2Props, PrepareC
              *  This is different for the 'old',v66, flow triggered by a 'threeDS2Challenge' action (which includes the threeds2InMDFlow)
              *  than for the new, v67, 'threeDS2' action
              */
-            const resolveDataFunction = this.props.useOriginalFlow ? createOldChallengeResolveData : createChallengeResolveData;
-            const data = resolveDataFunction(this.props.dataKey, resultObj.transStatus, this.props.paymentData);
 
-            this.props.onComplete(data); // (equals onAdditionalDetails - except for 3DS2InMDFlow)
+            if (this.state.challengeData.delegatedAuthenticationSDKInput && resultObj.transStatus === 'Y') {
+                this.setState({ status: 'delegatedAuthenticationEnrollment' });
```

Comment:
Since there are a few different status' it would be cleaner to create an enum rather than relying on strings.